alpaca/common/enums.py

```diff
@@ -10,6 +10,7 @@ class BaseURL(str, Enum):
     TRADING_LIVE = "https://api.alpaca.markets"
     DATA = "https://data.alpaca.markets"
     MARKET_DATA_STREAM = "wss://stream.data.alpaca.markets"
+    NEWS_DATA_STREAM = "wss://stream.data.alpaca.markets"
```

Comment:
This looks the same as `MARKET_DATA_STREAM` on the line above, why not just using that one?

Reply:
Hi @alessiocastrica, you are right. `NEWS_DATA_STREAM` does not need to be there since `MARKET_DATA_STREAM` already points to the same endpoint. However, my initial thoughts were modularity and consistency, but after some thought it makes sense to only have one endpoint for market data in general.

Reply:
https://github.com/alpacahq/alpaca-py/pull/369/commits/35b298fe0e9c1612dc41755cfa0dbbf415af8ebe